### PR TITLE
Terra-Icon - Switch to WDIO

### PR DESCRIPTION
**Pull request by saedar**
### Summary
This change is part of our ongoing efforts to migrate off of Nightwatch and onto WDIO testing.

Relates to #1302

### CODE REVIEW

File: packages/terra-icon/tests/wdio/icon-spec.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+describe('Icon', () => {
+  before(() => browser.setViewportSize(Terra.viewports('medium')[0]));
+  describe('Default', () => {
+    before(() => browser.url('/#/raw/tests/icon/icon-default'));
+
+    Terra.should.beAccessible();
+    Terra.should.matchScreenshot();
+  });
+
+  describe('Append', () => {
```
Comment (bjankord) [inline packages/terra-icon/tests/wdio/icon-spec.js]: This test can be removed as it is covered by Jest snapshot test.

File: packages/terra-icon/tests/wdio/icon-spec.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    before(() => browser.url('/#/raw/tests/icon/icon-append-class'));
+
+    Terra.should.beAccessible();
+    Terra.should.matchScreenshot();
+  });
+
+  describe('Aria', () => {
```
Comment (bjankord) [inline packages/terra-icon/tests/wdio/icon-spec.js]: This test can be removed as it is covered by Jest snapshot test.

File: packages/terra-icon/tests/wdio/icon-spec.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+  describe('HeightWidth', () => {
+    before(() => browser.url('/#/raw/tests/icon/icon-height-width'));
+
+    Terra.should.beAccessible();
+    Terra.should.matchScreenshot();
+  });
+
+  describe('Bidi', () => {
+    before(() => browser.url('/#/raw/tests/icon/icon-bidi'));
```
Comment (bjankord) [inline packages/terra-icon/tests/wdio/icon-spec.js]: Need to add `dir="rtl"` to the [div here](https://github.com/cerner/terra-core/blob/master/packages/terra-icon/tests/nightwatch/components/IconBidi.jsx#L7) so the `isBidi` prop behavior is visible in the screenshot.

File: packages/terra-icon/tests/wdio/icon-spec.js (hunk 1 above)
Comment (saedar) [reply packages/terra-icon/tests/wdio/icon-spec.js]: https://github.com/cerner/terra-core/pull/1530/commits/8e1e43bda772216f1638a0621580584836218f73

File: packages/terra-icon/tests/wdio/icon-spec.js (hunk 2 above)
Comment (saedar) [reply packages/terra-icon/tests/wdio/icon-spec.js]: https://github.com/cerner/terra-core/pull/1530/commits/8e1e43bda772216f1638a0621580584836218f73

File: packages/terra-icon/tests/wdio/icon-spec.js (hunk 3 above)
Comment (saedar) [reply packages/terra-icon/tests/wdio/icon-spec.js]: Brett did this in a commit of his own.

File: packages/terra-date-time-picker/translations/de.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
-{
-  "Terra.dateTimePicker.timeClarification.title": "Klärung der Zeit",
-  "Terra.dateTimePicker.timeClarification.message": "Die ausgewählte Zeit liegt während der Umstellung von Sommer- auf Standardzeit. Möchten Sie für die Zeit vor oder nach der Zeitumstellung dokumentieren?",
-  "Terra.dateTimePicker.timeClarification.button.daylightSaving": "Vor (Sommerzeit)",
-  "Terra.dateTimePicker.timeClarification.button.standardTime": "Nach (Standardzeit)"
+{
```
Comment (saedar) [inline packages/terra-date-time-picker/translations/de.json]: The only thing that changed in these translations was end of line encodings. No idea why my computer does this sometimes...

**Pull request merged**